lib/eslint.js

```diff
@@ -461,6 +462,7 @@ function getRuleReplacementMessage(ruleId) {
         var newRules = replacements.rules[ruleId];
         return "Rule \'" + ruleId + "\' was removed and replaced by: " + newRules.join(", ");
     }
+    return null;
```

Comment:
Because new `consistent-return` rule had been warning this function: `Expected to return a value at the end of this function.`